podman_cli/podman_images.sh

```diff
@@ -0,0 +1,236 @@
+#!/bin/sh
+
+# podman_images.sh demo script.
+# This script will demonstrate at an introductory level
+# the use of the podman images command.
+# Podman must be installed prior to running this script.
+
+ 
+# Setting up some colors for helping read the demo output.
+# Comment out any of the below to turn off that color.
+bold=$(tput bold)
+cyan=$(tput setaf 6)
+reset=$(tput sgr0)
+
+read_color() {
+    read -p "${bold}$1${reset}"
+}
+
+echo_color() {
+    echo "${cyan}$1${reset}"
+}
+
+setup() {
+    rpm -q podman >/dev/null
+    if [[ $? != 0 ]]; then
+	echo $0 requires the podman package to be installed
+	exit 1
+    fi
+    clear
+}
+
+intro() {
+    echo_color "\`podman images\` Demo"
+    echo
+    echo_color "Available at https://github.com/containers/Demos/podman_cli/podman_images.sh"
+    echo
+}
+
+version() {
+   
+    echo_color "First check the Podman version"
+    echo
+    read_color "podman version"
+    echo
+    podman version
+    echo
+
+    echo
+    read -p "Enter to continue"
+    clear
+}
+
+podman_pull_images() {
+    echo_color "Let's create our very first container image"
+    echo
+    read_color "podman pull alpine"
+    podman pull alpine
+
+    echo
+    echo_color "Let's look at the image"
+    echo
+    read_color "podman images"
+    podman images
+
+    echo
+    echo_color "Let's create a busybox and nginx container image"
+    echo
+    read_color "podman pull busybox"
+    podman pull busybox 
+    echo
+    read_color "podman pull nginx:latest"
+    podman pull nginx:latest
+
+    echo
+    echo_color "Let's look at the images"
+    echo
+    read_color "podman images"
+    podman images
+
+    echo
+    read -p "Enter to continue"
+    clear
+}
+
+
+podman_from_dockerfile() {
+
+    /bin/cat <<- "EOF" > ./Dockerfile.hello
+FROM alpine
+RUN apk add python3
+ADD HelloFromContainer.py /home
+WORKDIR HOME
+CMD ["python3","/home/HelloFromContainer.py"]
+EOF
+    echo
+    echo_color "Create an image from a Dockerfile and run the container"
+    echo_color "Let's first look at our Dockerfile"
+    echo
+    read_color "cat ./Dockerfile.hello"
+    cat ./Dockerfile.hello
+
+    /bin/cat <<- "EOF" > ./HelloFromContainer.py
```

Comment:
I would move this up before the echo lines, since users might go do a different window to examine this file.

Reply:
done